### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by arctouch-ismaelstahelin**
Couldn't find any major problem. It's a standard Angular app. 
- for using material design
- for the number of dependencies, which I'm not sure were all used in the project.

Please find additional comments in the "Files changed" section.

### CODE REVIEW

File: app/scripts/app.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ angular.module('FloripaRoute', ['ngRoute', 'ngAnimate', 'ngAria', 'ngMaterial',
             .primaryPalette('green');
     })
     .config(function ($httpProvider) {
+        //good use of interceptors
         $httpProvider.interceptors.push('authenticationInterceptor');
-		$httpProvider.interceptors.push('loadingInterceptor');
```
Comment (arctouch-ismaelstahelin) [inline app/scripts/app.js]: Why you decided to use interceptors? What are the advantages of using it?

**Comment by arctouch-ismaelstahelin** [comment]
What's your primary programming language of choice?
Which others you have experience with?
Why did you choose build the project using HTML5?

File: package.json [hunk 2]
```diff
@@ -30,6 +31,6 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "grunt test"
+    "test": "node_modules/.bin/grunt test"
```
Comment (arctouch-ismaelstahelin) [inline package.json]: There's no "test" grunt task defined.

File: app/scripts/app.js (hunk 1 above)
Comment (rauffer) [inline app/scripts/app.js]: I'm a lazy developer and I prefer to write code once. =]
Interceptors make the maintenance easier, preventing duplicated code.

Comment (arctouch-ismaelstahelin) [inline app/scripts/app.js]: Ok, that sounds like a good answer to me.
Thanks.

File: package.json (hunk 2 above)
Comment (rauffer) [inline package.json]: I'm sorry for the late. Busy days...
I'm a Java developer with deep experience developing web applications.
Javascript is an other language I have experience.

This is a small project, with simple requirements. HTML5 and Javascript can satisfy them pretty well.
Besides that, I had little time develop it, so I chose one of languages which I have expecience with.

**Pull request closed without merging**